content/docs/handling-events.md

```diff
@@ -1,43 +1,46 @@
 ---
 id: handling-events
-title: Handling Events
+title: Omgaan met Events
 permalink: docs/handling-events.html
 prev: state-and-lifecycle.html
 next: conditional-rendering.html
 redirect_from:
   - "docs/events-ko-KR.html"
 ---
 
-Handling events with React elements is very similar to handling events on DOM elements. There are some syntactic differences:
+Omgaan met events met React-elementen lijkt sterk op omgaan met events met DOM-elementen.
+Er zijn enkele syntactische verschillen:
```

Comment:
```suggestion
Maar, er zijn enkele syntactische verschillen:
```

Or add the word "Echter". We're making a 'tegenstelling' in this sentence compared to the previous, so it's proper to make that clear in the syntax as well.

Reply:
True, although the English version should technically have a "but" or "however" as well.

Reply:
I agree, but the English language is more forgiving in this, while Dutch is less so. So I think it's better to make it explicit, especially if we're also targetting less experiences devs.